lighttpd.conf

```diff
@@ -0,0 +1,58 @@
+server.modules += ( "mod_access",
+                    "mod_auth",
+                    "mod_fastcgi",
+                    "mod_alias",
+                    "mod_rewrite",
+                    "mod_openssl",
+                    "mod_accesslog",
+                    "mod_dirlisting")
+
+server.feature-flags = ( "server.network-mptcp" => "enable" )
+server.use-ipv6 = "enable"
+
+$SERVER["socket"] == "[::]:443" {
+    ssl.engine = "enable"
+    ssl.pemfile = "/etc/letsencrypt/live/check.mptcp.dev/fullchain.pem"
```

Comment:
`check.mptcp.dev` should be replaced.

What about what I suggested before in the chat, by using `YOUR_URL`, then people have to run:
```
sed -i "s#YOUR_URL#${url}#g" lighttpd.conf
```

(to be added in the README file)